app/controllers/transactions.py

```diff
@@ -43,3 +44,61 @@ def post(self):
 
         except Exception as err:
             return dict(status='fail', message=str(err)), 500
+
+    
+    @jwt_required
+    def get(self):
+
+        current_user_id = get_jwt_identity()
+        current_user_roles = get_jwt_claims()['roles']
+
+        transaction_schema = TransactionRecordSchema(many=True)
+
+        transaction = TransactionRecord.find_all()
```

Comment:
@e-ian this bit left me confused as to whether this is an admin endpoint where they can see all transactions or a regular user's endpoint where we would need to filter by user_id so that we only get that user's transactions.

Reply:
@MutegekiHenry look at the entire file, look at line 65

Reply:
I believe these are admin endpoints.

Reply:
They were actually user endpoints.

Reply:
@MutegekiHenry changes updated

Reply:
Cool.